### DISCUSSION TITLE
posts/2022/06/using-icloud-mail-custom-domain-following-up/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#16] [Announcements]
# posts/2022/06/using-icloud-mail-custom-domain-following-up/

What happened when I moved my main email address to iCloud Mail.

https://www.brycewray.com/posts/2022/06/using-icloud-mail-custom-domain-following-up/

**Comment by ScootRay** [comment]
IFastmail’s handy implementation of aliases and multiple domains into one box (I consolidated multiple email accounts into one while keeping each unique domain/identity) is what’s holding me back from migrating to iCloud. 

I’m curious if this tool (https://imapsync.lamiral.info/X/) would have helped; I had to use it to transfer emails to Fastmail since my old host was terribly slow. 

Thanks for sharing your journey into the iCloud!

Ray

**Reply by brycewray** [reply]
@ScootRay I didn’t know of that tool until now, so couldn’t say; however, would assume the iCloud Mail server-side speed issues I observed in Apple Mail’s Connection Doctor UI would’ve affected its use, too.

As for the alias/domain “goodies” in FastMail: in time, I came not to need them as much, so my use case obviously is less complicated than yours. I suspect that, eventually, Apple will step up and **try** to provide some of these capabilities, but won’t be in any great hurry about it.
